### ISSUE TITLE
Open picker with already selected pictures

### ISSUE THREAD

**Issue by djleonskennedy** [ivpusic/react-native-image-crop-picker#1295]
### Version
- react-native-image-crop-picker: ^0.31.1
- react-native v0.62.2

### Platform
- iOS

### Expected behaviour
Have ability to open picker with already selected pictures in multiple mode

### Actual behaviour
Not possible to get picker opened with selected pictures, all pictures should be reselected again

### Steps to reproduce
set picker with multiple mode, for example we have 3 pictures on phone

1. open two pictures, component get links back

2. what to add one to selection

3. no possibility to set selected

### Attachments

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by hainguyentien** [comment]
Any temporary solution for this, please?

**Comment by djleonskennedy** [comment]
@hainguyentien no, we solved this issue by explaining to customer that it's not possible yet )
